bot/src/main/java/de/chojo/gamejam/data/wrapper/team/JamTeam.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *     SPDX-License-Identifier: AGPL-3.0-only
+ *
+ *     Copyright (C) 2022 DevCord Team and Contributor
+ */
+
+package de.chojo.gamejam.data.wrapper.team;
+
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.Role;
+import net.dv8tion.jda.api.entities.TextChannel;
+import net.dv8tion.jda.api.entities.VoiceChannel;
+
+public record JamTeam(int id, String name, long leader, long roleId, long textChannelId,
+                      long voiceChannelId) {
+
+    public static JamTeam create(String name, Member leader, Role role, TextChannel textChannel, VoiceChannel voiceChannel) {
+        return new JamTeam(-1,
+                name,
+                leader.getIdLong(),
+                role.getIdLong(),
+                textChannel.getIdLong(),
+                voiceChannel.getIdLong());
+    }
```

Comment:
This is not necessary, just create an overloaded constructor

Reply:
Overloaded constructors are an anti pattern tho.
This is a factory method to create new not registered Teams and is used for registration of new teams.

Reply:
Since the default fields is an id, that is probably auto generated it think this method is just useless. 
```java
new JamTeam(name, leader, role, textChannel, voiceChannel)
```
has the same meaning for me and are you shure overloaded constructors are an anti pattern?